estuary/utils/story.py

```diff
@@ -0,0 +1,468 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import abc
+from neomodel import db
+
+from estuary.error import ValidationError
+from estuary.models.koji import ContainerKojiBuild, KojiBuild, ModuleKojiBuild
+from estuary.models.bugzilla import BugzillaBug
+from estuary.models.distgit import DistGitCommit
+from estuary.models.errata import Advisory, ContainerAdvisory
+from estuary.models.freshmaker import FreshmakerEvent
+
+
+class BaseStoryManager(object):
+    """A class containing utility methods to create a story for an artifact."""
+
+    @staticmethod
+    def get_story_manager(item, item_id, config, limit=False):
```

Comment:
I don't think you need the parameter `item_id` because `item.id` will get you this value.